websites/G/galaxy/presence.ts

```diff
@@ -0,0 +1,462 @@
+					recent: "browsing the most recently updated games",
+					playtime: "browsing the most grinded games",
+					new: "browsing the newest games",
+					random: "browsing random games",
+				},
+				category = path[1],
+				categoryName = categories[category];
+
+			presenceData.details = categoryName || "browsing for games";
+			presenceData.buttons = [
+				{
+					label: "explore games",
+					url: makeURL(`/explore/${category}`),
+				},
+			];
+			break;
+		}
+		// Other pages
+		case "signup": {
+			presenceData.details = "signing up";
+			break;
+		}
+		case "request": {
+			const gameName = document.querySelector<HTMLInputElement>(
+				'div[data-label="game name"] > input'
+			)?.value;
+
+			presenceData.details = "requesting a game";
+			if (gameName) presenceData.state = `"${gameName}"`;
+			presenceData.buttons = [
+				{
+					label: "request game",
+					url: makeURL("/request"),
+				},
+			];
+			break;
+		}
+		case "stats": {
+			presenceData.details = "viewing site statistics";
+			presenceData.buttons = [
+				{
+					label: "view stats",
+					url: makeURL("/stats"),
+				},
+			];
+			break;
+		}
+		case "cluster": {
+			presenceData.details = "trying out galaxy cluster";
+			presenceData.buttons = [
+				{
+					label: "galaxy cluster",
+					url: makeURL("/cluster"),
+				},
+			];
+			break;
+		}
+		case "dev": {
+			presenceData.details = "becoming a developer";
+			presenceData.buttons = [
+				{
+					label: "view page",
+					url: makeURL("/dev"),
+				},
+			];
+			break;
+		}
+		case "docs": {
+			presenceData.details = "reading the documentation";
+			presenceData.buttons = [
+				{
+					label: "read the docs",
+					url: makeURL(`${path.map(p => `/${p}`).join("")}`),
+				},
+			];
+			break;
+		}
+		case "about": {
+			presenceData.details = "reading about galaxy";
+			presenceData.buttons = [
+				{
+					label: "view page",
+					url: makeURL("/about"),
+				},
+			];
+			break;
+		}
+		case "rules": {
+			presenceData.details = "reviewing the site rules";
+			presenceData.buttons = [
+				{
+					label: "view page",
+					url: makeURL("/rules"),
+				},
+			];
+			break;
+		}
+		case "terms": {
+			presenceData.details = "reading the terms of service";
+			presenceData.buttons = [
+				{
+					label: "view page",
+					url: makeURL("/terms"),
+				},
+			];
+			break;
+		}
+		case "privacy": {
+			presenceData.details = "reading the privacy policy";
+			presenceData.buttons = [
+				{
+					label: "view page",
+					url: makeURL("/privacy"),
+				},
+			];
+			break;
+		}
+		case "cookies": {
+			presenceData.details = "reading the cookie policy";
+			presenceData.buttons = [
+				{
+					label: "view page",
+					url: makeURL("/cookies"),
+				},
+			];
+			break;
+		}
+		case "banned": {
+			presenceData.details = "banned";
+			break;
+		}
+		case "pogger": {
+			// https://galaxy.click/pogger
+			presenceData.buttons = [
+				{
+					label: "pogger button",
+					url: "https://save418.com",
```

Comment:
why

Reply:
In the site, there is a chance for a link to a [secret page](https://galaxy.click/pogger) to appear in the footer. I thought it would be fun to include a similar button to the one that appears in that page.

Reply:
The site owner has requested that I remove the button